src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex.GlobalAttributeInfo.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.FindSymbols
+{
+    internal partial class SyntaxTreeIndex
+    {
+        private readonly struct GlobalAttributeInfo
+        {
+            private static readonly GlobalAttributeInfo s_empty = new GlobalAttributeInfo(ImmutableDictionary<string, ImmutableArray<int>>.Empty);
+
+            /// <summary>
+            /// Map from string literals inside global attributes to their positions inside the syntax tree.
+            /// </summary>
```

Comment:
can you still explain what the key is here.  i still dont' get it.

Reply:
No longer relevant with the new logic.